src/components/misc/importer/ImporterColumnHead.jsx

```diff
@@ -36,6 +38,11 @@ export default class ImporterColumnHead extends React.Component {
             'person_tag': col('colOptions.complex.tag'),
         };
 
+        const FIELD_OPTIONS = this.props.fieldTypes.items.reduce((obj, field) => {
+            obj['person_field.' + field.data.id + '.' + field.data.type] = field.data.title;
```

Comment:
This seems unnecessarily complex. Maybe this is what's causing that bug you were experiencing? I'm guessing this is there to be able to send `field_type` along with the `config` but as I've mentioned elsewhere I don't think we should be including `field_type` there. Which should allow for simplifying this.